### DISCUSSION TITLE
Hi everyone

### DISCUSSION THREAD

**Discussion by soum1t** [community/community#138011] [GitHub Education]
It's been few days since I was approved. When am I going to receive student benefits ??

**Comment by Shriharsh-Deshmukh** [comment]
Hi @soumit4646,
I also had a similar experience. After being approved, I received the student benefits within a day. It didn't take long for them to reflect on my profile— you'll notice a pro icon appear, indicating that your benefits are active. If you haven't seen any changes yet, it might just be a short delay, but they should come through soon!

**Comment by MainakSil** [comment]
When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application).

**Comment by andrefqms** [comment]
hi @soumit4646 

usually it's around 3-5 days, but depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks;

If the problem persists I'd suggest you launch a ticket over to Github Education's Support regarding this matter. You can do so [here](https://support.github.com/contact/education).

**Comment by davevad93** [comment]
Hi @soumit4646 ,

What's the status of your application?

If the label is green and says "`Congrats! Your request for @soumit4646 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and says "`Congrats! Your request for @soumit4646 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

**Comment by Ehab98** [comment]
hello @davevad93  and @andrefqms  , i have this 
We're still processing your application. Once this is complete you will have access to the student developer pack offers.

i was having the student pack from last year , what should I do , if I do anything it redirect me to pricing page 

can you guide me please ?

**Reply by davevad93** [reply]
Hey @Ehab98 , if you get redirected to the pricing page you should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue. You aren't the only one who has experienced it, as you can see in this solved discussion #136400.
